doc/once.txt

```diff
@@ -0,0 +1,13 @@
+* _.once(f)
```

Comment:
Why `once` rather than `lazy`? Allowing arguments seems a wee bit confusing too, but eh.

Reply:
same names as lodash

Reply:
https://lodash.com/docs/4.17.10#once

Reply:
I agree with squid, `lazy` seems like a better name. We aren't lodash